### PR TITLE
Feat #671 Change login flow

### LINKED ISSUE #671: Change login flow
### Prerequisites

- [x] I have written a descriptive issue title
- [x] I have verified that I am running the latest version of the COMET-WEB
- [x] I have searched [open](https://github.com/RHEAGROUP/COMET-WEB-Community-Edition/issues) and [closed](https://github.com/RHEAGROUP/COMET-WEB-Community-Edition/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description

Remove the request to open a model after the login, the user should be able to use the webapp without opening a model if not required

see https://www.figma.com/proto/HExiPH4WmdOhBSv4bXdhrL/comet-interface?page-id=0%3A1&amp%3Bnode-id=238-4003&amp%3Bviewport=1860%2C-247%2C0.45&amp%3Bt=iOKJLPpLjd3oXODj-1&amp%3Bscaling=scale-down&amp%3Bcontent-scaling=fixed&amp%3Bstarting-point-node-id=97%3A2004&utm_source=link-unfurl&utm_medium=msft&utm_content=HExiPH4WmdOhBSv4bXdhrL&utm_product_type=design&type=design&node-id=97-2004&starting-point-node-id=97%3A2004

### Logs

  - [x] There are no logs regarding the issue
  - [x] I have attached a log file or included the relevant section in this issue  
  
### System Configuration

- COMET version:
  - [ ] COMET-WEB: 
  - [ ] Other: 
- Web Services:
  - [ ] COMET Web Services version:

### PR DESCRIPTION
**Pull request by joao4all**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/RHEAGROUP/COMET-WEB-Community-Edition/pulls) open
- [x] I have verified that I am following the COMET-WEB [code style guidelines](https://raw.githubusercontent.com/RHEAGROUP/COMET-WEB-Community-Edition/master/.github/CONTRIBUTING.md)
- [x] I have provided test coverage for my change (where applicable)

### Description
Closes #671 Change login flow

### CODE REVIEW

File: COMETwebapp/ViewModels/Pages/TabsViewModel.cs [hunk 1]
```diff
@@ -84,7 +86,11 @@ public TabsViewModel(ISessionService sessionService, IServiceProvider servicePro
         public TabbedApplication SelectedApplication
         {
             get => this.selectedApplication;
-            set => this.RaiseAndSetIfChanged(ref this.selectedApplication, value);
```
Comment (antoineatstariongroup) [inline COMETwebapp/ViewModels/Pages/TabsViewModel.cs:87]: Why this change?

Comment (joao4all) [reply COMETwebapp/ViewModels/Pages/TabsViewModel.cs:87]: Because if I select an application that is already selected, the method InitializeViewModelBasedOnApplication won't be triggered. This is an issue atm, because the opentab component sets the current application for the tab, and if the application selected in the form is already selected, nothing happens inside the tabvm. This flow will likely to be changed when we implement support for multiple models in this page.

**Review by antoineatstariongroup — APPROVED** [review]
